deploy-eks/fb-editor-chart/templates/deployment.yaml

```diff
@@ -256,6 +256,36 @@ spec:
               secretKeyRef:
                 name: fb-editor-secrets-{{ .Values.environmentName }}
                 key: slack_publish_webhook
+          - name: AWS_S3_ACCESS_KEY_ID_{{ .Values.environmentName | upper }}_DEV
+            valueFrom:
+              secretKeyRef:
+                name: s3-service-metadata-{{ .Values.environmentName }}-dev
+                key: aws_s3_access_key_id
+          - name: AWS_S3_SECRET_ACCESS_KEY_{{ .Values.environmentName | upper }}_DEV
+            valueFrom:
+              secretKeyRef:
+                name: s3-service-metadata-{{ .Values.environmentName }}-dev
+                key: aws_s3_secret_access_key
+          - name: AWS_S3_BUCKET_{{ .Values.environmentName | upper }}_DEV
+            valueFrom:
+              secretKeyRef:
+                name: s3-service-metadata-{{ .Values.environmentName }}-dev
+                key: aws_s3_bucket
+          - name: AWS_S3_ACCESS_KEY_ID_{{ .Values.environmentName | upper }}_DEV
+            valueFrom:
+              secretKeyRef:
+                name: s3-service-metadata-{{ .Values.environmentName }}-production
+                key: aws_s3_access_key_id_{{ .Values.environmentName }}_production
```

Comment:
just checking this is meant to have the prefix `{{ .Values.environmentName }}_production`, asking as the corresponding value for dev is a little different.